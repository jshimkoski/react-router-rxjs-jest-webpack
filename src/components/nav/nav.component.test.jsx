import React from 'react';
import Nav from './nav.component';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';

describe('Nav', () => {
  let component;
  beforeEach(() => {
    component = renderer.create(
      <MemoryRouter>
        <Nav/>
      </MemoryRouter>
    ).toJSON();
  });

  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});