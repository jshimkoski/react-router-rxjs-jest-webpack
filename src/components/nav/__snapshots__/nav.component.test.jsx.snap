// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<nav
  className="nav"
>
  <img
    src="test-file-stub"
  />
  <ul>
    <li>
      <a
        className="active "
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className={undefined}
        href="/about"
        onClick={[Function]}
        style={undefined}
      >
        About
      </a>
    </li>
  </ul>
</nav>
`;
